Sumo Stuff/hello.nod.xml

```diff
@@ -1,9 +1,20 @@
 <nodes>
-    <node id="1" x="-100.0" y="0.0" />
```

Comment:
I doubled the road lengths so that the entire road wouldn't get filled up too soon.